developer/deploying/platform.md

```diff
@@ -0,0 +1,179 @@
+# Reaction Platform
+
+Reaction Commerce offers a managed deployment option, the [Reaction Platform](http://getrxn.io/reaction-platform).
+
+The `reaction-cli` incorporates functionality for any team to deploy Reaction to multiple environments. [Visit reactioncommerce.com](http://getrxn.io/reaction-platform) to get a demo or request an **invite token**.
+
+1.  [Request invite token](http://getrxn.io/reaction-platform).
+2.  Register local environment
+3.  Create, add, and publish SSH keys
+4.  Create application environment
+5.  Deploy application
+
+## Configuration
+
+As a user of the **Reaction Platform**, you'll receive an email containing a "invite token" from the Reaction Platform API ("Launchdock"). **Launchdock** is the name of our internal  orchestration management platform.
+
+You will be asked for your [invite token](http://getrxn.io/reaction-platform) when you use the [reaction-cli](http://getrxn.io/reaction-cli) to register your local Reaction environment with the Reaction Platform API.
+
+### register
+
+`reaction register` to create keys locally before configuring your application environment. `reaction login` authenticates you to the Reaction Platform and syncronizes platform services.
```

Comment:
This is sort of inaccurate, we have an [issue to remove the manual requirement](https://github.com/reactioncommerce/launchdock/issues/66), until then `register` saving the token.